setup.py

```diff
@@ -71,18 +75,18 @@ def run_tests(self):
         pytest.main(self.test_args)
 
 
-cmdclass = versioneer.get_cmdclass()
-cmdclass.update({"test": OpenSCMTest})
+CMDCLASS = versioneer.get_cmdclass()
+CMDCLASS.update({"test": OpenSCMTest})
 
 setup(
-    name=PACKAGE_NAME,
+    name="openscm",
     version=versioneer.get_version(),
-    description=DESCRIPTION,
-    long_description=README_TEXT,
+    description="A unifying interface for Simple Climate Models",
+    long_description="\n".join(README_LINES),
     long_description_content_type="text/x-rst",
-    author=AUTHOR,
-    author_email=EMAIL,
-    url=URL,
+    author="Robert Gieseke, Jared Lewis, Zeb Nicholls, Sven Willner",
```

Comment:
Sorry @swillner I really don't understand why we've moved all these away from being defined in one place at the top of the file?

Reply:
matter of taste, i prefer keeping definition and usage close if its used only once. @rgieseke you decide ;)

Reply:
If you prefer the constants at the top of the file then use constants for each of these? Also for classifiers?

Reply:
that would be my preference but ultimately it's no big deal so up to @rgieseke

Reply:
As @rgieseke pointed me to the way pandas does it (https://github.com/pandas-dev/pandas/blob/master/setup.py#L200), I bow my head in light of the better argument ;)

Reply:
I'd check what Numpy, Pandas do ...

https://github.com/pandas-dev/pandas/blob/master/setup.py#L200

https://github.com/numpy/numpy/blob/master/setup.py#L36

Numpy has a mix, but classifiers separately, I think it's useful to have everything settable in one place only. Makes it easier to use the rest as a template for the next project.